lib/pbio/include/pbdrv/sound.h

```diff
@@ -17,6 +17,15 @@
 
 #if PBDRV_CONFIG_SOUND
 
+/**
+ * Starts playing a square wave until pbdrv_sound_stop() is called.
+ *
+ * @param [in]  frequency           The frequency of the wave in Hz
+ * @param [in]  sample_attenuator   The amplitude of the wave in sample LSBs
```

Comment:
```suggestion
 * @param [in]  sample_attenuator   The normalized attenuation to apply to get the requested volume.
```

We can't control the amplitude of the wave on EV3 should make the language a bit more general.